RoeiVerenigingWPF/Pages/ListBoats.xaml

```diff
@@ -0,0 +1,68 @@
+﻿<Page x:Class="RoeiVerenigingWPF.Pages.ListBoats"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:RoeiVerenigingWPF.Pages"
+      mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="800"
+      Title="ListBoats">
+
+    <Grid>
+
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="1*" MaxHeight="100px"/>
+            <RowDefinition Height="50" MaxHeight="100px"/>
+            <RowDefinition Height="1*" />
+
+
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="1*" />
+            <ColumnDefinition Width="1*"/>
+
+            <ColumnDefinition Width="1*" />
+        </Grid.ColumnDefinitions>
+
+        <!-- Titel -->
+        <Label Content="Boten" FontSize="30" Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2"/>
+
+        <Grid Grid.Row="1" Grid.Column="0" Grid.ColumnSpan="3" Grid.RowSpan="4" MaxHeight="750px">
+            <ScrollViewer VerticalScrollBarVisibility="Visible" HorizontalScrollBarVisibility="Disabled">
+                <ListView ItemsSource="{Binding boats}" Grid.IsSharedSizeScope="True" d:ItemsSource="{d:SampleData ItemCount=8}">
+                    <ListView.ItemsPanel>
+                        <ItemsPanelTemplate>
+                            <UniformGrid Columns="2"/>
+                        </ItemsPanelTemplate>
+                    </ListView.ItemsPanel>
+                    <ListView.ItemTemplate>
+                        <DataTemplate>
+                            <Grid  Margin="50 20 50 0" Width="650" Height="200" >
+                                <Button Click="Button_Click" CommandParameter="{Binding Id}">
+                                    <!-- Define your item layout here -->
+                                    
+                                    <Border BorderBrush="Black" BorderThickness="1" Width="650" Height="200">
+                                        <Grid>
+                                            <Image Source="/Images/Image_not_available.png" MaxHeight="100px" MaxWidth="200px" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                            <StackPanel Margin="110 0 5 0">
+                                                <TextBlock Text="{Binding Id, StringFormat='Boot: {0}'}" VerticalAlignment="Center" HorizontalAlignment="Left" FontSize="25"/>
+                                                <TextBlock Text="{Binding Seats, StringFormat='Aantal man: {0}'}" VerticalAlignment="Center" HorizontalAlignment="Right"/>
+                                                <TextBlock Text="{Binding CaptainSeat, StringFormat='Stuurman? {0}'}" VerticalAlignment="Center" HorizontalAlignment="Right"/>
```

Comment:
Hier staat true of false. kan dit niet beter iets zijn van "stuurman positie aanwezig" en dit alleen laten zien als dit er is?